ci_config.json

```diff
@@ -176,6 +175,15 @@
       "cpp_std=c++17"
     ]
   },
+  "protobuf" : {
+    "_comment": ["- PDB can't handle protobuf.dll and protobuf.exe"],
+    "build_on": {
+      "windows": false
```

Comment:
You forgot to remove this, so it still isn't being tested on Windows. I assume it *should* work now, yes...